extensions/interop/interop.cs

```diff
@@ -324,6 +339,13 @@ public enum LIBFLAGS : short
         LIBFLAG_FHIDDEN = (short)4,
         LIBFLAG_FRESTRICTED = (short)1,
     }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(64), Inherited=false, AllowMultiple=false)]
+    public sealed partial class ManagedToNativeComInteropStubAttribute : System.Attribute
```

Comment:
Don't think of things being interop.cs to mean anything. I just move everything that I take out of netstandard into some extension so I can see what I transferred.